workspaces/analytics/plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts

```diff
@@ -121,6 +125,9 @@ export class GoogleAnalytics4 implements AnalyticsApi, NewAnalyticsApi {
     const identity =
       config.getOptionalString('app.analytics.ga4.identity') || 'disabled';
     const debug = config.getOptionalBoolean('app.analytics.ga4.debug') ?? false;
+    const cookieFlags =
+      config.getOptionalString('app.analytics.ga4.cookieFlags') ??
+      'SameSite=Lax;Secure';
```

Comment:
Will it work if Backstage isn't served over `https`? Shouldn't this be default to:

```suggestion
      'SameSite=Lax';
```

Reply:
done